docs/advanced/input_files/input-main.md

```diff
@@ -711,26 +711,26 @@ calculations.
 
 - **Type**: String
 - **Description**: Choose the basis set.
-  - pw: Using plane-wave basis set only.
-  - lcao_in_pw: Expand the localized atomic set in plane-wave basis.
-  - lcao: Using localized atomic orbital sets.
+  - **pw**: Using plane-wave basis set only.
+  - **lcao_in_pw**: Expand the localized atomic set in plane-wave basis.
+  - **lcao**: Using localized atomic orbital sets.
 - **Default**: pw
 
 ### ks_solver
 
 - **Type**: String
-- **Description**: It's about the choice of diagonalization methods for the Hamiltonian matrix expanded in a certain basis set.
+- **Description**: Choose the diagonalization methods for the Hamiltonian matrix expanded in a certain basis set.
 
   For plane-wave basis,
 
-  - cg: cg method.
-  - dav: the Davidson algorithm.
+  - **cg**: cg method.
+  - **dav**: the Davidson algorithm.
 
   For atomic orbitals basis,
 
-  - genelpa: This method should be used if you choose localized orbitals.
-  - scalapack-gvx: scalapack can also be used for localized orbitals.
-  - cusolver: this method needs building with the cusolver component for lcao and at least one gpu is available.
+  - **genelpa**: This method should be used if you choose localized orbitals.
+  - **scalapack-gvx**: scalapack can also be used for localized orbitals.
+  - **cusolver**: this method needs building with the cusolver component for lcao and at least one gpu is available.
```

Comment:
This option is unable in this version.

Reply:
If this option is useless, maybe it's better to remove them from the codes, and then delete this option in the doc.

Reply:
> If this option is useless, maybe it's better to remove them from the codes, and then delete this option in the doc.

This option is not useless, but can not be used in this version, it can be discussed whether fix or delete.

Reply:
It's done.